rewrite-yaml/src/main/java/org/openrewrite/yaml/MergeYamlVisitor.java

```diff
@@ -317,19 +321,26 @@ private Yaml.Sequence mergeSequence(Yaml.Sequence s1, Yaml.Sequence s2, P p, Cur
         }
     }
 
-    private String grabBeforeFirstLineBreak(Yaml.Mapping.Entry entry) {
-        String[] parts = LINE_BREAK.split(entry.getPrefix());
-        return parts.length > 0 ? parts[0] : "";
-    }
-
-    private String grabAfterFirstLineBreak(Yaml.Mapping.Entry entry) {
-        String[] parts = LINE_BREAK.split(entry.getPrefix());
-        return parts.length > 1 ? String.join(linebreak(), Arrays.copyOfRange(parts, 1, parts.length)) : "";
-    }
-
     private Yaml.Scalar mergeScalar(Yaml.Scalar y1, Yaml.Scalar y2) {
         String s1 = y1.getValue();
         String s2 = y2.getValue();
         return !s1.equals(s2) && !acceptTheirs ? y1.withValue(s2) : y1;
     }
+
+    public String substringOfBeforeFirstLineBreak(String s) {
+        String[] lines = LINE_BREAK.split(s);
+        return lines.length > 0 ? lines[0] : "";
+    }
+
+    public String substringOfAfterFirstLineBreak(String s) {
+        String[] lines = LINE_BREAK.split(s);
+        return lines.length > 1 ? String.join(linebreak(), Arrays.copyOfRange(lines, 1, lines.length)) : "";
+    }
```

Comment:
Nitpicking here; but these do not have to be public right? And they are the same as the methods above? Can we minimize the diff and exposed API please?